docs/deployment/local-deployment.rst

```diff
@@ -175,9 +172,89 @@ We now need to update our ``docker-compose.yml`` file, we add a new volume for t
             volumes:
                 - ./mongo/data:/data/db
 
-        blazegraph:
-            image: metaphacts/blazegraph-basic:2.2.0-20160908.003514-6
+        graphdb:
+            image: ontotext/graphdb:10.7.6
             ports:
-                - 8080:8080
+                - 7200:7200
             volumes:
-                - ./blazegraph:/blazegraph-data
+                - ./graphdb:/opt/graphdb/home
```

Comment:
Just wondering:

Why do the examples use [bind mounts][1] instead of [named volumes][2]? 

Is there a specific reason for this, or is it just a legacy thing?

[1]: https://docs.docker.com/engine/storage/bind-mounts/
[2]: https://docs.docker.com/engine/storage/volumes/

Reply:
Legacy I assume, and it makes the setup more self-contained.

Reply:
If there is no specific reason, maybe good to upgrade to [named volumes][2] as well:

>***Volumes are the preferred mechanism*** for persisting data generated by and used by Docker containers. While [bind mounts](https://docs.docker.com/engine/storage/bind-mounts/) are dependent on the directory structure and OS of the host machine, volumes are completely managed by Docker. Volumes have several advantages over bind mounts: [...]

and from [bind mounts][1]:

> Bind mounts have been around since the early days of Docker. Bind mounts have limited functionality compared to volumes. [...] If you are developing new Docker applications, consider using [named volumes](https://docs.docker.com/engine/storage/volumes/) instead. You can't use Docker CLI commands to directly manage bind mounts.

Would look like this:

```yaml
# ...
  mongo:
    image: mongo:4.0.12
    volumes:
      - mongodb:/data/db

  graphdb:
    image: ontotext/graphdb:10.7.6
    ports:
      - 7200:7200
    volumes:
      - graphdb:/opt/graphdb/home

volumes:
  graphdb:
  mongodb:
```

[1]: https://docs.docker.com/engine/storage/bind-mounts/
[2]: https://docs.docker.com/engine/storage/volumes/

Reply:
Sounds like a good addition. You can append to this PR by pushing to this branch, or create a new PR because these changes are getting way out of scope.

Reply:
Maybe a new PR would be fitting, as changing the volume type could have consequences for users with *existing* data. Those data would need to be migrated from the *existing* bind mount location to the *new* named volume. 

Then again, a similar issue would probably arise for users needing to migrate from an existing blazegraph to a new graphdb?

Perhaps a separate section on db migration?